ckanext/canada/view.py

```diff
@@ -201,13 +240,26 @@ class CanadaUserRegisterView(UserRegisterView):
     def post(self):
         params = parse_params(request.form)
         email = params.get('email', '')
+        if isinstance(email, list):
+            email = email[0]
```

Comment:
All of these could be replaced like this:
```suggestion
        email = request.form.get('email')
```
MultiDict returns the first item with `[]` and `get()` when multiple are passed.

Reply:
Updated to do it how CKAN Core views do it:
```
clean_dict(unflatten(tuplize_dict(parse_params(request.form))))
```